qa/coccinelle/malloc-error-check.cocci

```diff
@@ -58,6 +58,8 @@ FAIL_IF(x == NULL)
 FAIL_IF(unlikely(x == NULL))
 |
 FAIL_IF_NULL(x)
+|
+return x
```

Comment:
Works with
```
return x;
```
otherwise cocci fails to load this file.

Reply:
Cocci cannot however detect if the caller does indeed check that value, so this could miss some things?

Reply:
> return x;

changed and force-pushed

Reply:
> Cocci cannot however detect if the caller does indeed check that value, so this could miss some things?

Sure